geonode/tests/integration.py

```diff
@@ -59,6 +59,48 @@
 
 logging.getLogger("south").setLevel(logging.INFO)
 
+"""
+ HOW TO RUN THE TESTS
+ --------------------
+ (https://github.com/GeoNode/geonode/blob/master/docs/tutorials/devel/testing.txt)
+
+ 1)
+  (https://github.com/GeoNode/geonode/blob/master/docs/tutorials/devel/envsetup/paver.txt)
+
+  $ paver setup
+
+   1a. If using a PostgreSQL DB
```

Comment:
Good point, we are not using postrgresql for tests but we should since we use it in production